src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -12,7 +11,10 @@ namespace System.Linq
     // NOTE: DO NOT DELETE THE FOLLOWING DEBUG VIEW TYPES.
     // Although it might be tempting due to them not be referenced anywhere in this library,
     // Visual Studio currently depends on their existence to enable the "Results" view in 
-    // watch windows.
+    // watch windows.  Specifically:
+    // - It creates the debug view by name.
+    // - It looks for the empty exception by name.
+    // - It uses the exception's Empty property by name.
```

Comment:
My understanding is that once it's created by name, it's like any other debug view and has the same constraints as any other debug view.  I didn't include those; I was just listing the ones special to this type.